Cannot compile arm multilibs if both thumb and arm mode cannot be disabled

1. ct-ng version: 1.25.0_rc1
2. Host OS: Docker Ubuntu 20.04.4
3. <details close>
    <summary>.config (click show details)</summary>

    ```
    #
    # Automatically generated file; DO NOT EDIT.
    # crosstool-NG 1.25.0_rc1 Configuration
    #
    CT_CONFIGURE_has_static_link=y
    CT_CONFIGURE_has_cxx11=y
    CT_CONFIGURE_has_wget=y
    CT_CONFIGURE_has_make_3_81_or_newer=y
    CT_CONFIGURE_has_make_4_0_or_newer=y
    CT_CONFIGURE_has_libtool_2_4_or_newer=y
    CT_CONFIGURE_has_libtoolize_2_4_or_newer=y
    CT_CONFIGURE_has_autoconf_2_65_or_newer=y
    CT_CONFIGURE_has_autoreconf_2_65_or_newer=y
    CT_CONFIGURE_has_automake_1_15_or_newer=y
    CT_CONFIGURE_has_gnu_m4_1_4_12_or_newer=y
    CT_CONFIGURE_has_python_3_4_or_newer=y
    CT_CONFIGURE_has_bison_2_7_or_newer=y
    CT_CONFIGURE_has_python=y
    CT_CONFIGURE_has_git=y
    CT_CONFIGURE_has_md5sum=y
    CT_CONFIGURE_has_sha1sum=y
    CT_CONFIGURE_has_sha256sum=y
    CT_CONFIGURE_has_sha512sum=y
    CT_CONFIGURE_has_install_with_strip_program=y
    CT_VERSION="1.25.0_rc1"
    CT_VCHECK=""
    CT_CONFIG_VERSION_ENV="4"
    CT_CONFIG_VERSION_CURRENT="4"
    CT_CONFIG_VERSION="4"
    CT_MODULES=y
    
    #
    # Paths and misc options
    #
    
    #
    # crosstool-NG behavior
    #
    # CT_OBSOLETE is not set
    CT_EXPERIMENTAL=y
    CT_ALLOW_BUILD_AS_ROOT=y
    CT_ALLOW_BUILD_AS_ROOT_SURE=y
    # CT_DEBUG_CT is not set
    
    #
    # Paths
    #
    CT_LOCAL_TARBALLS_DIR="${HOME}/src"
    CT_SAVE_TARBALLS=y
    # CT_TARBALLS_BUILDROOT_LAYOUT is not set
    CT_WORK_DIR="${CT_TOP_DIR}/.build"
    CT_BUILD_TOP_DIR="${CT_WORK_DIR:-${CT_TOP_DIR}/.build}/${CT_HOST:+HOST-${CT_HOST}/}${CT_TARGET}"
    CT_BUILD_DIR="${CT_BUILD_TOP_DIR}/build"
    CT_PREFIX_DIR="${CT_PREFIX:-${HOME}/x-tools}/${CT_HOST:+HOST-${CT_HOST}/}${CT_TARGET}"
    CT_RM_RF_PREFIX_DIR=y
    CT_REMOVE_DOCS=y
    CT_INSTALL_LICENSES=y
    CT_PREFIX_DIR_RO=y
    CT_STRIP_HOST_TOOLCHAIN_EXECUTABLES=y
    # CT_STRIP_TARGET_TOOLCHAIN_EXECUTABLES is not set
    
    #
    # Downloading
    #
    CT_DOWNLOAD_AGENT_WGET=y
    # CT_DOWNLOAD_AGENT_NONE is not set
    # CT_FORBID_DOWNLOAD is not set
    # CT_FORCE_DOWNLOAD is not set
    CT_CONNECT_TIMEOUT=10
    CT_DOWNLOAD_WGET_OPTIONS="--passive-ftp --tries=3 -nc --progress=dot:binary"
    # CT_ONLY_DOWNLOAD is not set
    # CT_USE_MIRROR is not set
    CT_VERIFY_DOWNLOAD_DIGEST=y
    CT_VERIFY_DOWNLOAD_DIGEST_SHA512=y
    # CT_VERIFY_DOWNLOAD_DIGEST_SHA256 is not set
    # CT_VERIFY_DOWNLOAD_DIGEST_SHA1 is not set
    # CT_VERIFY_DOWNLOAD_DIGEST_MD5 is not set
    CT_VERIFY_DOWNLOAD_DIGEST_ALG="sha512"
    # CT_VERIFY_DOWNLOAD_SIGNATURE is not set
    
    #
    # Extracting
    #
    # CT_FORCE_EXTRACT is not set
    CT_OVERRIDE_CONFIG_GUESS_SUB=y
    # CT_ONLY_EXTRACT is not set
    CT_PATCH_BUNDLED=y
    # CT_PATCH_LOCAL is not set
    # CT_PATCH_BUNDLED_LOCAL is not set
    # CT_PATCH_LOCAL_BUNDLED is not set
    # CT_PATCH_NONE is not set
    CT_PATCH_ORDER="bundled"
    
    #
    # Build behavior
    #
    CT_PARALLEL_JOBS=0
    CT_LOAD=""
    CT_USE_PIPES=y
    CT_EXTRA_CFLAGS_FOR_BUILD=""
    CT_EXTRA_CXXFLAGS_FOR_BUILD=""
    CT_EXTRA_LDFLAGS_FOR_BUILD=""
    CT_EXTRA_CFLAGS_FOR_HOST=""
    CT_EXTRA_LDFLAGS_FOR_HOST=""
    # CT_CONFIG_SHELL_SH is not set
    # CT_CONFIG_SHELL_ASH is not set
    CT_CONFIG_SHELL_BASH=y
    # CT_CONFIG_SHELL_CUSTOM is not set
    CT_CONFIG_SHELL="${bash}"
    
    #
    # Logging
    #
    # CT_LOG_ERROR is not set
    # CT_LOG_WARN is not set
    # CT_LOG_INFO is not set
    CT_LOG_EXTRA=y
    # CT_LOG_ALL is not set
    # CT_LOG_DEBUG is not set
    CT_LOG_LEVEL_MAX="EXTRA"
    # CT_LOG_SEE_TOOLS_WARN is not set
    CT_LOG_PROGRESS_BAR=y
    CT_LOG_TO_FILE=y
    # CT_LOG_FILE_COMPRESS is not set
    # end of Paths and misc options
    
    #
    # Target options
    #
    # CT_ARCH_ALPHA is not set
    # CT_ARCH_ARC is not set
    CT_ARCH_ARM=y
    # CT_ARCH_AVR is not set
    # CT_ARCH_C6X is not set
    # CT_ARCH_M68K is not set
    # CT_ARCH_MICROBLAZE is not set
    # CT_ARCH_MIPS is not set
    # CT_ARCH_MOXIE is not set
    # CT_ARCH_MSP430 is not set
    # CT_ARCH_NIOS2 is not set
    # CT_ARCH_POWERPC is not set
    # CT_ARCH_PRU is not set
    # CT_ARCH_RISCV is not set
    # CT_ARCH_S390 is not set
    # CT_ARCH_SH is not set
    # CT_ARCH_SPARC is not set
    # CT_ARCH_X86 is not set
    # CT_ARCH_XTENSA is not set
    CT_ARCH="arm"
    CT_ARCH_CHOICE_KSYM="ARM"
    CT_ARCH_CPU=""
    CT_ARCH_TUNE=""
    CT_ARCH_ARM_SHOW=y
    
    #
    # Options for arm
    #
    CT_ARCH_ARM_PKG_KSYM=""
    CT_ARCH_ARM_MODE="thumb"
    # CT_ARCH_ARM_MODE_ARM is not set
    CT_ARCH_ARM_MODE_THUMB=y
    # CT_ARCH_ARM_INTERWORKING is not set
    CT_ARCH_ARM_EABI_FORCE=y
    CT_ARCH_ARM_EABI=y
    CT_ALL_ARCH_CHOICES="ALPHA ARC ARM AVR C6X M68K MICROBLAZE MIPS MOXIE MSP430 NIOS2 POWERPC PRU RISCV S390 SH SPARC X86 XTENSA"
    CT_ARCH_SUFFIX=""
    # CT_OMIT_TARGET_VENDOR is not set
    
    #
    # Generic target options
    #
    CT_MULTILIB=y
    # CT_DEMULTILIB is not set
    CT_ARCH_SUPPORTS_BOTH_MMU=y
    CT_ARCH_DEFAULT_HAS_MMU=y
    # CT_ARCH_USE_MMU is not set
    CT_ARCH_SUPPORTS_FLAT_FORMAT=y
    CT_ARCH_SUPPORTS_EITHER_ENDIAN=y
    CT_ARCH_DEFAULT_LE=y
    # CT_ARCH_BE is not set
    CT_ARCH_LE=y
    CT_ARCH_ENDIAN="little"
    CT_ARCH_SUPPORTS_32=y
    CT_ARCH_SUPPORTS_64=y
    CT_ARCH_DEFAULT_32=y
    CT_ARCH_BITNESS=32
    CT_ARCH_32=y
    # CT_ARCH_64 is not set
    
    #
    # Target optimisations
    #
    CT_ARCH_SUPPORTS_WITH_ARCH=y
    CT_ARCH_SUPPORTS_WITH_CPU=y
    CT_ARCH_SUPPORTS_WITH_TUNE=y
    CT_ARCH_SUPPORTS_WITH_FLOAT=y
    CT_ARCH_SUPPORTS_WITH_FPU=y
    CT_ARCH_SUPPORTS_SOFTFP=y
    CT_ARCH_EXCLUSIVE_WITH_CPU=y
    CT_ARCH_ARCH=""
    CT_ARCH_FPU=""
    CT_ARCH_FLOAT_AUTO=y
    # CT_ARCH_FLOAT_HW is not set
    # CT_ARCH_FLOAT_SOFTFP is not set
    # CT_ARCH_FLOAT_SW is not set
    CT_TARGET_CFLAGS=""
    CT_TARGET_LDFLAGS=""
    CT_ARCH_FLOAT="auto"
    # end of Target options
    
    #
    # Toolchain options
    #
    
    #
    # General toolchain options
    #
    CT_WANTS_STATIC_LINK=y
    CT_WANTS_STATIC_LINK_CXX=y
    # CT_STATIC_TOOLCHAIN is not set
    CT_SHOW_CT_VERSION=y
    CT_TOOLCHAIN_PKGVERSION=""
    CT_TOOLCHAIN_BUGURL=""
    
    #
    # Tuple completion and aliasing
    #
    CT_TARGET_VENDOR="none"
    CT_TARGET_ALIAS_SED_EXPR=""
    CT_TARGET_ALIAS=""
    
    #
    # Toolchain type
    #
    # CT_NATIVE is not set
    CT_CROSS=y
    # CT_CROSS_NATIVE is not set
    # CT_CANADIAN is not set
    CT_TOOLCHAIN_TYPE="cross"
    
    #
    # Build system
    #
    CT_BUILD=""
    CT_BUILD_PREFIX=""
    CT_BUILD_SUFFIX=""
    
    #
    # Misc options
    #
    # CT_TOOLCHAIN_ENABLE_NLS is not set
    # end of Toolchain options
    
    #
    # Operating System
    #
    CT_BARE_METAL=y
    CT_KERNEL_BARE_METAL=y
    # CT_KERNEL_LINUX is not set
    CT_KERNEL="bare-metal"
    CT_KERNEL_CHOICE_KSYM="BARE_METAL"
    CT_KERNEL_BARE_METAL_SHOW=y
    
    #
    # Options for bare-metal
    #
    CT_KERNEL_BARE_METAL_PKG_KSYM=""
    CT_ALL_KERNEL_CHOICES="BARE_METAL LINUX WINDOWS"
    
    #
    # Common kernel options
    #
    # end of Operating System
    
    #
    # Binary utilities
    #
    # CT_ARCH_BINFMT_ELF is not set
    CT_ARCH_BINFMT_FLAT=y
    # CT_ARCH_BINFMT_FDPIC is not set
    CT_BINUTILS_BINUTILS=y
    CT_BINUTILS="binutils"
    CT_BINUTILS_CHOICE_KSYM="BINUTILS"
    CT_BINUTILS_BINUTILS_SHOW=y
    
    #
    # Options for binutils
    #
    CT_BINUTILS_BINUTILS_PKG_KSYM="BINUTILS"
    CT_BINUTILS_DIR_NAME="binutils"
    CT_BINUTILS_USE_GNU=y
    # CT_BINUTILS_USE_LINARO is not set
    # CT_BINUTILS_USE_ORACLE is not set
    CT_BINUTILS_USE="BINUTILS"
    CT_BINUTILS_PKG_NAME="binutils"
    CT_BINUTILS_SRC_RELEASE=y
    # CT_BINUTILS_SRC_DEVEL is not set
    # CT_BINUTILS_SRC_CUSTOM is not set
    CT_BINUTILS_PATCH_GLOBAL=y
    # CT_BINUTILS_PATCH_BUNDLED is not set
    # CT_BINUTILS_PATCH_LOCAL is not set
    # CT_BINUTILS_PATCH_BUNDLED_LOCAL is not set
    # CT_BINUTILS_PATCH_LOCAL_BUNDLED is not set
    # CT_BINUTILS_PATCH_NONE is not set
    CT_BINUTILS_PATCH_ORDER="global"
    CT_BINUTILS_V_2_38=y
    # CT_BINUTILS_V_2_37 is not set
    # CT_BINUTILS_V_2_36 is not set
    # CT_BINUTILS_V_2_35 is not set
    # CT_BINUTILS_V_2_34 is not set
    # CT_BINUTILS_V_2_33 is not set
    # CT_BINUTILS_V_2_32 is not set
    # CT_BINUTILS_V_2_31 is not set
    # CT_BINUTILS_V_2_30 is not set
    # CT_BINUTILS_V_2_29 is not set
    # CT_BINUTILS_V_2_28 is not set
    # CT_BINUTILS_V_2_27 is not set
    # CT_BINUTILS_V_2_26 is not set
    CT_BINUTILS_VERSION="2.38"
    CT_BINUTILS_MIRRORS="$(CT_Mirrors GNU binutils) $(CT_Mirrors sourceware binutils/releases)"
    CT_BINUTILS_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_BINUTILS_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_BINUTILS_ARCHIVE_FORMATS=".tar.xz .tar.bz2 .tar.gz"
    CT_BINUTILS_SIGNATURE_FORMAT="packed/.sig"
    CT_BINUTILS_later_than_2_30=y
    CT_BINUTILS_2_30_or_later=y
    CT_BINUTILS_later_than_2_27=y
    CT_BINUTILS_2_27_or_later=y
    CT_BINUTILS_later_than_2_26=y
    CT_BINUTILS_2_26_or_later=y
    
    #
    # GNU binutils
    #
    CT_BINUTILS_GOLD_SUPPORTS_ARCH=y
    CT_BINUTILS_GOLD_SUPPORT=y
    CT_BINUTILS_FORCE_LD_BFD_ONLY=y
    CT_BINUTILS_LINKER_LD=y
    CT_BINUTILS_LINKERS_LIST="ld"
    CT_BINUTILS_LINKER_DEFAULT="bfd"
    CT_BINUTILS_PLUGINS=y
    CT_BINUTILS_RELRO=m
    CT_BINUTILS_DETERMINISTIC_ARCHIVES=y
    CT_BINUTILS_EXTRA_CONFIG_ARRAY=""
    
    #
    # elf2flt
    #
    CT_ELF2FLT_REQUIRES=y
    CT_ELF2FLT_DIR_NAME="elf2flt"
    CT_ELF2FLT_PKG_NAME="elf2flt"
    CT_ELF2FLT_SRC_DEVEL=y
    # CT_ELF2FLT_SRC_CUSTOM is not set
    CT_ELF2FLT_DEVEL_VCS_git=y
    # CT_ELF2FLT_DEVEL_VCS_svn is not set
    # CT_ELF2FLT_DEVEL_VCS_hg is not set
    # CT_ELF2FLT_DEVEL_VCS_cvs is not set
    CT_ELF2FLT_DEVEL_VCS="git"
    CT_ELF2FLT_DEVEL_URL="https://github.com/uclinux-dev/elf2flt.git"
    CT_ELF2FLT_DEVEL_BRANCH=""
    CT_ELF2FLT_DEVEL_REVISION="453398f917d167f8c308c8f997270c48ae8f8b12"
    CT_ELF2FLT_DEVEL_SUBDIR=""
    CT_ELF2FLT_DEVEL_BOOTSTRAP=""
    CT_ELF2FLT_PATCH_GLOBAL=y
    # CT_ELF2FLT_PATCH_BUNDLED is not set
    # CT_ELF2FLT_PATCH_LOCAL is not set
    # CT_ELF2FLT_PATCH_BUNDLED_LOCAL is not set
    # CT_ELF2FLT_PATCH_LOCAL_BUNDLED is not set
    # CT_ELF2FLT_PATCH_NONE is not set
    CT_ELF2FLT_PATCH_ORDER="global"
    CT_ELF2FLT_VERSION="unknown"
    CT_ELF2FLT_EXTRA_CONFIG_ARRAY=""
    CT_ALL_BINUTILS_CHOICES="BINUTILS"
    # end of Binary utilities
    
    #
    # C-library
    #
    CT_LIBC_NEWLIB=y
    # CT_LIBC_NONE is not set
    CT_LIBC="newlib"
    CT_LIBC_CHOICE_KSYM="NEWLIB"
    CT_THREADS="none"
    CT_LIBC_NEWLIB_SHOW=y
    
    #
    # Options for newlib
    #
    CT_LIBC_NEWLIB_PKG_KSYM="NEWLIB"
    CT_NEWLIB_DIR_NAME="newlib"
    CT_NEWLIB_PKG_NAME="newlib"
    CT_NEWLIB_SRC_RELEASE=y
    # CT_NEWLIB_SRC_DEVEL is not set
    # CT_NEWLIB_SRC_CUSTOM is not set
    CT_NEWLIB_PATCH_GLOBAL=y
    # CT_NEWLIB_PATCH_BUNDLED is not set
    # CT_NEWLIB_PATCH_LOCAL is not set
    # CT_NEWLIB_PATCH_BUNDLED_LOCAL is not set
    # CT_NEWLIB_PATCH_LOCAL_BUNDLED is not set
    # CT_NEWLIB_PATCH_NONE is not set
    CT_NEWLIB_PATCH_ORDER="global"
    CT_NEWLIB_V_4_1=y
    # CT_NEWLIB_V_3_3 is not set
    # CT_NEWLIB_V_3_2 is not set
    # CT_NEWLIB_V_3_1 is not set
    # CT_NEWLIB_V_3_0 is not set
    # CT_NEWLIB_V_2_5 is not set
    CT_NEWLIB_VERSION="4.1.0"
    CT_NEWLIB_MIRRORS="ftp://sourceware.org/pub/newlib"
    CT_NEWLIB_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_NEWLIB_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_NEWLIB_ARCHIVE_FORMATS=".tar.gz"
    CT_NEWLIB_SIGNATURE_FORMAT=""
    CT_LIBC_NEWLIB_TARGET_CFLAGS=""
    # CT_LIBC_NEWLIB_IO_C99FMT is not set
    # CT_LIBC_NEWLIB_IO_LL is not set
    # CT_LIBC_NEWLIB_IO_FLOAT is not set
    # CT_LIBC_NEWLIB_IO_POS_ARGS is not set
    # CT_LIBC_NEWLIB_FVWRITE_IN_STREAMIO is not set
    # CT_LIBC_NEWLIB_UNBUF_STREAM_OPT is not set
    # CT_LIBC_NEWLIB_FSEEK_OPTIMIZATION is not set
    CT_LIBC_NEWLIB_DISABLE_SUPPLIED_SYSCALLS=y
    # CT_LIBC_NEWLIB_REGISTER_FINI is not set
    # CT_LIBC_NEWLIB_ATEXIT_DYNAMIC_ALLOC is not set
    # CT_LIBC_NEWLIB_GLOBAL_ATEXIT is not set
    CT_LIBC_NEWLIB_LITE_EXIT=y
    CT_LIBC_NEWLIB_REENT_SMALL=y
    # CT_LIBC_NEWLIB_MULTITHREAD is not set
    # CT_LIBC_NEWLIB_RETARGETABLE_LOCKING is not set
    CT_LIBC_NEWLIB_EXTRA_SECTIONS=y
    # CT_LIBC_NEWLIB_WIDE_ORIENT is not set
    CT_LIBC_NEWLIB_ENABLE_TARGET_OPTSPACE=y
    # CT_LIBC_NEWLIB_LTO is not set
    CT_LIBC_NEWLIB_NANO_MALLOC=y
    CT_LIBC_NEWLIB_NANO_FORMATTED_IO=y
    CT_LIBC_NEWLIB_EXTRA_CONFIG_ARRAY=""
    CT_ALL_LIBC_CHOICES="AVR_LIBC BIONIC GLIBC MINGW_W64 MOXIEBOX MUSL NEWLIB NONE UCLIBC_NG"
    CT_LIBC_SUPPORT_THREADS_NONE=y
    CT_LIBC_PROVIDES_CXA_ATEXIT=y
    
    #
    # Common C library options
    #
    CT_THREADS_NONE=y
    # end of C-library
    
    #
    # C compiler
    #
    CT_CC_CORE_NEEDED=y
    CT_CC_SUPPORT_CXX=y
    CT_CC_SUPPORT_FORTRAN=y
    CT_CC_SUPPORT_ADA=y
    CT_CC_SUPPORT_OBJC=y
    CT_CC_SUPPORT_OBJCXX=y
    CT_CC_SUPPORT_GOLANG=y
    CT_CC_GCC=y
    CT_CC="gcc"
    CT_CC_CHOICE_KSYM="GCC"
    CT_CC_GCC_SHOW=y
    
    #
    # Options for gcc
    #
    CT_CC_GCC_PKG_KSYM="GCC"
    CT_GCC_DIR_NAME="gcc"
    CT_GCC_USE_GNU=y
    # CT_GCC_USE_LINARO is not set
    CT_GCC_USE="GCC"
    CT_GCC_PKG_NAME="gcc"
    CT_GCC_SRC_RELEASE=y
    # CT_GCC_SRC_DEVEL is not set
    # CT_GCC_SRC_CUSTOM is not set
    CT_GCC_PATCH_GLOBAL=y
    # CT_GCC_PATCH_BUNDLED is not set
    # CT_GCC_PATCH_LOCAL is not set
    # CT_GCC_PATCH_BUNDLED_LOCAL is not set
    # CT_GCC_PATCH_LOCAL_BUNDLED is not set
    # CT_GCC_PATCH_NONE is not set
    CT_GCC_PATCH_ORDER="global"
    CT_GCC_V_11=y
    # CT_GCC_V_10 is not set
    # CT_GCC_V_9 is not set
    # CT_GCC_V_8 is not set
    # CT_GCC_V_7 is not set
    # CT_GCC_V_6 is not set
    # CT_GCC_V_5 is not set
    # CT_GCC_V_4_9 is not set
    CT_GCC_VERSION="11.2.0"
    CT_GCC_MIRRORS="$(CT_Mirrors GNU gcc/gcc-${CT_GCC_VERSION}) $(CT_Mirrors sourceware gcc/releases/gcc-${CT_GCC_VERSION})"
    CT_GCC_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_GCC_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_GCC_ARCHIVE_FORMATS=".tar.xz .tar.gz"
    CT_GCC_SIGNATURE_FORMAT=""
    CT_GCC_later_than_11=y
    CT_GCC_11_or_later=y
    CT_GCC_later_than_10=y
    CT_GCC_10_or_later=y
    CT_GCC_later_than_9=y
    CT_GCC_9_or_later=y
    CT_GCC_later_than_8=y
    CT_GCC_8_or_later=y
    CT_GCC_later_than_7=y
    CT_GCC_7_or_later=y
    CT_GCC_later_than_6=y
    CT_GCC_6_or_later=y
    CT_GCC_later_than_5=y
    CT_GCC_5_or_later=y
    CT_GCC_later_than_4_9=y
    CT_GCC_4_9_or_later=y
    CT_CC_GCC_ENABLE_PLUGINS=y
    CT_CC_GCC_HAS_LIBMPX=y
    CT_CC_GCC_ENABLE_CXX_FLAGS=""
    CT_CC_GCC_CORE_EXTRA_CONFIG_ARRAY=""
    CT_CC_GCC_EXTRA_CONFIG_ARRAY=""
    CT_CC_GCC_MULTILIB_LIST="rmprofile"
    CT_CC_GCC_STATIC_LIBSTDCXX=y
    # CT_CC_GCC_SYSTEM_ZLIB is not set
    # CT_CC_GCC_CONFIG_TLS is not set
    
    #
    # Optimisation features
    #
    CT_CC_GCC_USE_GRAPHITE=y
    CT_CC_GCC_USE_LTO=y
    CT_CC_GCC_LTO_ZSTD=m
    
    #
    # Settings for libraries running on target
    #
    # CT_CC_GCC_ENABLE_TARGET_OPTSPACE is not set
    # CT_CC_GCC_LIBMUDFLAP is not set
    # CT_CC_GCC_LIBSSP is not set
    # CT_CC_GCC_LIBQUADMATH is not set
    
    #
    # Misc. obscure options.
    #
    # CT_CC_CXA_ATEXIT is not set
    CT_CC_GCC_TM_CLONE_REGISTRY=m
    CT_CC_GCC_DISABLE_PCH=y
    # CT_CC_GCC_LDBL_128 is not set
    CT_CC_GCC_BUILD_ID=y
    CT_CC_GCC_LNK_HASH_STYLE_DEFAULT=y
    # CT_CC_GCC_LNK_HASH_STYLE_SYSV is not set
    # CT_CC_GCC_LNK_HASH_STYLE_GNU is not set
    # CT_CC_GCC_LNK_HASH_STYLE_BOTH is not set
    CT_CC_GCC_LNK_HASH_STYLE=""
    CT_CC_GCC_DEC_FLOATS_AUTO=y
    # CT_CC_GCC_DEC_FLOATS_BID is not set
    # CT_CC_GCC_DEC_FLOATS_DPD is not set
    # CT_CC_GCC_DEC_FLOATS_NO is not set
    CT_CC_GCC_DEC_FLOATS=""
    CT_ALL_CC_CHOICES="GCC"
    
    #
    # Additional supported languages:
    #
    CT_CC_LANG_CXX=y
    # CT_CC_LANG_FORTRAN is not set
    CT_CC_LANG_OTHERS=""
    # end of C compiler
    
    #
    # Debug facilities
    #
    # CT_DEBUG_GDB is not set
    # CT_DEBUG_LTRACE is not set
    # CT_DEBUG_STRACE is not set
    CT_ALL_DEBUG_CHOICES="DUMA GDB LTRACE STRACE"
    # end of Debug facilities
    
    #
    # Companion libraries
    #
    # CT_COMPLIBS_CHECK is not set
    # CT_COMP_LIBS_CLOOG is not set
    CT_COMP_LIBS_EXPAT=y
    CT_COMP_LIBS_EXPAT_PKG_KSYM="EXPAT"
    CT_EXPAT_DIR_NAME="expat"
    CT_EXPAT_PKG_NAME="expat"
    CT_EXPAT_SRC_RELEASE=y
    # CT_EXPAT_SRC_DEVEL is not set
    # CT_EXPAT_SRC_CUSTOM is not set
    CT_EXPAT_PATCH_GLOBAL=y
    # CT_EXPAT_PATCH_BUNDLED is not set
    # CT_EXPAT_PATCH_LOCAL is not set
    # CT_EXPAT_PATCH_BUNDLED_LOCAL is not set
    # CT_EXPAT_PATCH_LOCAL_BUNDLED is not set
    # CT_EXPAT_PATCH_NONE is not set
    CT_EXPAT_PATCH_ORDER="global"
    CT_EXPAT_V_2_4=y
    CT_EXPAT_VERSION="2.4.1"
    CT_EXPAT_MIRRORS="http://downloads.sourceforge.net/project/expat/expat/${CT_EXPAT_VERSION} https://github.com/libexpat/libexpat/releases/download/R_${CT_EXPAT_VERSION//./_}"
    CT_EXPAT_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_EXPAT_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_EXPAT_ARCHIVE_FORMATS=".tar.xz .tar.lz .tar.bz2 .tar.gz"
    CT_EXPAT_SIGNATURE_FORMAT=""
    # CT_COMP_LIBS_GETTEXT is not set
    CT_COMP_LIBS_GMP=y
    CT_COMP_LIBS_GMP_PKG_KSYM="GMP"
    CT_GMP_DIR_NAME="gmp"
    CT_GMP_PKG_NAME="gmp"
    CT_GMP_SRC_RELEASE=y
    # CT_GMP_SRC_DEVEL is not set
    # CT_GMP_SRC_CUSTOM is not set
    CT_GMP_PATCH_GLOBAL=y
    # CT_GMP_PATCH_BUNDLED is not set
    # CT_GMP_PATCH_LOCAL is not set
    # CT_GMP_PATCH_BUNDLED_LOCAL is not set
    # CT_GMP_PATCH_LOCAL_BUNDLED is not set
    # CT_GMP_PATCH_NONE is not set
    CT_GMP_PATCH_ORDER="global"
    CT_GMP_V_6_2=y
    # CT_GMP_V_6_1 is not set
    CT_GMP_VERSION="6.2.1"
    CT_GMP_MIRRORS="https://gmplib.org/download/gmp https://gmplib.org/download/gmp/archive $(CT_Mirrors GNU gmp)"
    CT_GMP_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_GMP_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_GMP_ARCHIVE_FORMATS=".tar.xz .tar.lz .tar.bz2"
    CT_GMP_SIGNATURE_FORMAT="packed/.sig"
    CT_COMP_LIBS_ISL=y
    CT_COMP_LIBS_ISL_PKG_KSYM="ISL"
    CT_ISL_DIR_NAME="isl"
    CT_ISL_PKG_NAME="isl"
    CT_ISL_SRC_RELEASE=y
    # CT_ISL_SRC_DEVEL is not set
    # CT_ISL_SRC_CUSTOM is not set
    CT_ISL_PATCH_GLOBAL=y
    # CT_ISL_PATCH_BUNDLED is not set
    # CT_ISL_PATCH_LOCAL is not set
    # CT_ISL_PATCH_BUNDLED_LOCAL is not set
    # CT_ISL_PATCH_LOCAL_BUNDLED is not set
    # CT_ISL_PATCH_NONE is not set
    CT_ISL_PATCH_ORDER="global"
    CT_ISL_V_0_24=y
    # CT_ISL_V_0_23 is not set
    # CT_ISL_V_0_22 is not set
    # CT_ISL_V_0_21 is not set
    # CT_ISL_V_0_20 is not set
    # CT_ISL_V_0_19 is not set
    # CT_ISL_V_0_18 is not set
    # CT_ISL_V_0_17 is not set
    # CT_ISL_V_0_16 is not set
    # CT_ISL_V_0_15 is not set
    CT_ISL_VERSION="0.24"
    CT_ISL_MIRRORS="https://libisl.sourceforge.io"
    CT_ISL_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_ISL_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_ISL_ARCHIVE_FORMATS=".tar.xz .tar.bz2 .tar.gz"
    CT_ISL_SIGNATURE_FORMAT=""
    CT_ISL_later_than_0_18=y
    CT_ISL_0_18_or_later=y
    CT_ISL_later_than_0_15=y
    CT_ISL_0_15_or_later=y
    # CT_COMP_LIBS_LIBELF is not set
    CT_COMP_LIBS_LIBICONV=y
    CT_COMP_LIBS_LIBICONV_PKG_KSYM="LIBICONV"
    CT_LIBICONV_DIR_NAME="libiconv"
    CT_LIBICONV_PKG_NAME="libiconv"
    CT_LIBICONV_SRC_RELEASE=y
    # CT_LIBICONV_SRC_DEVEL is not set
    # CT_LIBICONV_SRC_CUSTOM is not set
    CT_LIBICONV_PATCH_GLOBAL=y
    # CT_LIBICONV_PATCH_BUNDLED is not set
    # CT_LIBICONV_PATCH_LOCAL is not set
    # CT_LIBICONV_PATCH_BUNDLED_LOCAL is not set
    # CT_LIBICONV_PATCH_LOCAL_BUNDLED is not set
    # CT_LIBICONV_PATCH_NONE is not set
    CT_LIBICONV_PATCH_ORDER="global"
    CT_LIBICONV_V_1_16=y
    # CT_LIBICONV_V_1_15 is not set
    CT_LIBICONV_VERSION="1.16"
    CT_LIBICONV_MIRRORS="$(CT_Mirrors GNU libiconv)"
    CT_LIBICONV_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_LIBICONV_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_LIBICONV_ARCHIVE_FORMATS=".tar.gz"
    CT_LIBICONV_SIGNATURE_FORMAT="packed/.sig"
    CT_COMP_LIBS_MPC=y
    CT_COMP_LIBS_MPC_PKG_KSYM="MPC"
    CT_MPC_DIR_NAME="mpc"
    CT_MPC_PKG_NAME="mpc"
    CT_MPC_SRC_RELEASE=y
    # CT_MPC_SRC_DEVEL is not set
    # CT_MPC_SRC_CUSTOM is not set
    CT_MPC_PATCH_GLOBAL=y
    # CT_MPC_PATCH_BUNDLED is not set
    # CT_MPC_PATCH_LOCAL is not set
    # CT_MPC_PATCH_BUNDLED_LOCAL is not set
    # CT_MPC_PATCH_LOCAL_BUNDLED is not set
    # CT_MPC_PATCH_NONE is not set
    CT_MPC_PATCH_ORDER="global"
    CT_MPC_V_1_2=y
    # CT_MPC_V_1_1 is not set
    # CT_MPC_V_1_0 is not set
    CT_MPC_VERSION="1.2.1"
    CT_MPC_MIRRORS="http://www.multiprecision.org/downloads $(CT_Mirrors GNU mpc)"
    CT_MPC_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_MPC_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_MPC_ARCHIVE_FORMATS=".tar.gz"
    CT_MPC_SIGNATURE_FORMAT="packed/.sig"
    CT_MPC_later_than_1_1_0=y
    CT_MPC_1_1_0_or_later=y
    CT_COMP_LIBS_MPFR=y
    CT_COMP_LIBS_MPFR_PKG_KSYM="MPFR"
    CT_MPFR_DIR_NAME="mpfr"
    CT_MPFR_PKG_NAME="mpfr"
    CT_MPFR_SRC_RELEASE=y
    # CT_MPFR_SRC_DEVEL is not set
    # CT_MPFR_SRC_CUSTOM is not set
    CT_MPFR_PATCH_GLOBAL=y
    # CT_MPFR_PATCH_BUNDLED is not set
    # CT_MPFR_PATCH_LOCAL is not set
    # CT_MPFR_PATCH_BUNDLED_LOCAL is not set
    # CT_MPFR_PATCH_LOCAL_BUNDLED is not set
    # CT_MPFR_PATCH_NONE is not set
    CT_MPFR_PATCH_ORDER="global"
    CT_MPFR_V_4_1=y
    # CT_MPFR_V_4_0 is not set
    # CT_MPFR_V_3_1 is not set
    CT_MPFR_VERSION="4.1.0"
    CT_MPFR_MIRRORS="http://www.mpfr.org/mpfr-${CT_MPFR_VERSION} $(CT_Mirrors GNU mpfr)"
    CT_MPFR_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_MPFR_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_MPFR_ARCHIVE_FORMATS=".tar.xz .tar.bz2 .tar.gz .zip"
    CT_MPFR_SIGNATURE_FORMAT="packed/.asc"
    CT_MPFR_later_than_4_0_0=y
    CT_MPFR_4_0_0_or_later=y
    CT_COMP_LIBS_NCURSES=y
    CT_COMP_LIBS_NCURSES_PKG_KSYM="NCURSES"
    CT_NCURSES_DIR_NAME="ncurses"
    CT_NCURSES_PKG_NAME="ncurses"
    CT_NCURSES_SRC_RELEASE=y
    # CT_NCURSES_SRC_DEVEL is not set
    # CT_NCURSES_SRC_CUSTOM is not set
    CT_NCURSES_PATCH_GLOBAL=y
    # CT_NCURSES_PATCH_BUNDLED is not set
    # CT_NCURSES_PATCH_LOCAL is not set
    # CT_NCURSES_PATCH_BUNDLED_LOCAL is not set
    # CT_NCURSES_PATCH_LOCAL_BUNDLED is not set
    # CT_NCURSES_PATCH_NONE is not set
    CT_NCURSES_PATCH_ORDER="global"
    CT_NCURSES_V_6_2=y
    # CT_NCURSES_V_6_1 is not set
    # CT_NCURSES_V_6_0 is not set
    CT_NCURSES_VERSION="6.2"
    CT_NCURSES_MIRRORS="https://invisible-mirror.net/archives/ncurses $(CT_Mirrors GNU ncurses)"
    CT_NCURSES_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_NCURSES_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_NCURSES_ARCHIVE_FORMATS=".tar.gz"
    CT_NCURSES_SIGNATURE_FORMAT="packed/.sig"
    CT_NCURSES_NEW_ABI=y
    CT_NCURSES_HOST_CONFIG_ARGS=""
    CT_NCURSES_HOST_DISABLE_DB=y
    CT_NCURSES_HOST_FALLBACKS="linux,xterm,xterm-color,xterm-256color,vt100"
    CT_NCURSES_TARGET_CONFIG_ARGS=""
    # CT_NCURSES_TARGET_DISABLE_DB is not set
    CT_NCURSES_TARGET_FALLBACKS=""
    # CT_COMP_LIBS_NEWLIB_NANO is not set
    CT_COMP_LIBS_ZLIB=y
    CT_COMP_LIBS_ZLIB_PKG_KSYM="ZLIB"
    CT_ZLIB_DIR_NAME="zlib"
    CT_ZLIB_PKG_NAME="zlib"
    CT_ZLIB_SRC_RELEASE=y
    # CT_ZLIB_SRC_DEVEL is not set
    # CT_ZLIB_SRC_CUSTOM is not set
    CT_ZLIB_PATCH_GLOBAL=y
    # CT_ZLIB_PATCH_BUNDLED is not set
    # CT_ZLIB_PATCH_LOCAL is not set
    # CT_ZLIB_PATCH_BUNDLED_LOCAL is not set
    # CT_ZLIB_PATCH_LOCAL_BUNDLED is not set
    # CT_ZLIB_PATCH_NONE is not set
    CT_ZLIB_PATCH_ORDER="global"
    CT_ZLIB_V_1_2_11=y
    CT_ZLIB_VERSION="1.2.11"
    CT_ZLIB_MIRRORS="http://downloads.sourceforge.net/project/libpng/zlib/${CT_ZLIB_VERSION} https://www.zlib.net/"
    CT_ZLIB_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
    CT_ZLIB_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
    CT_ZLIB_ARCHIVE_FORMATS=".tar.xz .tar.gz"
    CT_ZLIB_SIGNATURE_FORMAT="packed/.asc"
    CT_ALL_COMP_LIBS_CHOICES="CLOOG EXPAT GETTEXT GMP GNUPRUMCU ISL LIBELF LIBICONV MPC MPFR NCURSES NEWLIB_NANO PICOLIBC ZLIB"
    CT_LIBICONV_NEEDED=y
    # CT_GETTEXT_NEEDED is not set
    CT_GMP_NEEDED=y
    CT_MPFR_NEEDED=y
    CT_ISL_NEEDED=y
    CT_MPC_NEEDED=y
    CT_ZLIB_NEEDED=y
    CT_LIBICONV=y
    CT_GMP=y
    CT_MPFR=y
    CT_ISL=y
    CT_MPC=y
    CT_ZLIB=y
    # end of Companion libraries
    
    #
    # Companion tools
    #
    # CT_COMP_TOOLS_FOR_HOST is not set
    # CT_COMP_TOOLS_AUTOCONF is not set
    # CT_COMP_TOOLS_AUTOMAKE is not set
    # CT_COMP_TOOLS_BISON is not set
    # CT_COMP_TOOLS_DTC is not set
    # CT_COMP_TOOLS_LIBTOOL is not set
    # CT_COMP_TOOLS_M4 is not set
    # CT_COMP_TOOLS_MAKE is not set
    CT_ALL_COMP_TOOLS_CHOICES="AUTOCONF AUTOMAKE BISON DTC LIBTOOL M4 MAKE"
    # end of Companion tools
    
    #
    # Test suite
    #
    # CT_TEST_SUITE_GCC is not set
    # end of Test suite
    ```

    </details>

4. [build.log](https://filebin.net/0u1nu4584or3mzbr) (This is not a build.1 log, as I think this bug has nothing to do with the log, but if it's really necessary, I'll upload it later)

Error log highlight:
```
[INFO ]  Installing core C gcc compiler
[EXTRA]    Configuring core C gcc compiler
[ERROR]    Error: You cannot use any of --with-arch/cpu/fpu/float/mode with --with-multilib-list=rmprofile
[ERROR]    make[1]: *** [Makefile:4352: configure-gcc] Error 1
```
According to the `--with-multilib-list=list` option of the [GCC Configuration Manual](https://gcc.gnu.org/install/configure.html), 
> _list_ is a comma separated list of **aprofile** and **rmprofile** to build multilibs for A or R and M architecture profiles respectively. .... The special value **default** is also accepted and is equivalent to omitting the option, i.e., only the default run-time library will be enabled. 

if I understand correctly, I can only choose three options for the `list` part if I want to configure the armxx architecture, `aprofile`, `rmprofile`, or `default`(or nothing). However, the error above occurred when I set the `CT_CC_GCC_MULTILIB_LIST=rmprofile` and built the GCC.

According to the error message and the `rmprofile` makefile [code](https://github.com/gcc-mirror/gcc/blob/3734527dfa0d10a50aee2f088d37320000fd65bf/gcc/config/arm/t-rmprofile#L23), it requires that all the arch/cpu/fpu/float/mode cannot be set when configuring multilibs. But the `ARCH_ARM_MODE` cannot be disabled from menuconfig, meaning I cannot build multilibs with this standard configuring method. So I commented out the `CT_ARCH_ARM_MODE="thumb"` line in the config file and was able to build multilibs-featured GCC.

<details><summary>The worked config</summary>

```
#
# Automatically generated file; DO NOT EDIT.
# crosstool-NG 1.25.0_rc1 Configuration
#
CT_CONFIGURE_has_static_link=y
CT_CONFIGURE_has_cxx11=y
CT_CONFIGURE_has_wget=y
CT_CONFIGURE_has_make_3_81_or_newer=y
CT_CONFIGURE_has_make_4_0_or_newer=y
CT_CONFIGURE_has_libtool_2_4_or_newer=y
CT_CONFIGURE_has_libtoolize_2_4_or_newer=y
CT_CONFIGURE_has_autoconf_2_65_or_newer=y
CT_CONFIGURE_has_autoreconf_2_65_or_newer=y
CT_CONFIGURE_has_automake_1_15_or_newer=y
CT_CONFIGURE_has_gnu_m4_1_4_12_or_newer=y
CT_CONFIGURE_has_python_3_4_or_newer=y
CT_CONFIGURE_has_bison_2_7_or_newer=y
CT_CONFIGURE_has_python=y
CT_CONFIGURE_has_git=y
CT_CONFIGURE_has_md5sum=y
CT_CONFIGURE_has_sha1sum=y
CT_CONFIGURE_has_sha256sum=y
CT_CONFIGURE_has_sha512sum=y
CT_CONFIGURE_has_install_with_strip_program=y
CT_VERSION="1.25.0_rc1"
CT_VCHECK=""
CT_CONFIG_VERSION_ENV="4"
CT_CONFIG_VERSION_CURRENT="4"
CT_CONFIG_VERSION="4"
CT_MODULES=y

#
# Paths and misc options
#

#
# crosstool-NG behavior
#
# CT_OBSOLETE is not set
CT_EXPERIMENTAL=y
CT_ALLOW_BUILD_AS_ROOT=y
CT_ALLOW_BUILD_AS_ROOT_SURE=y
# CT_DEBUG_CT is not set

#
# Paths
#
CT_LOCAL_TARBALLS_DIR="${HOME}/src"
CT_SAVE_TARBALLS=y
# CT_TARBALLS_BUILDROOT_LAYOUT is not set
CT_WORK_DIR="${CT_TOP_DIR}/.build"
CT_BUILD_TOP_DIR="${CT_WORK_DIR:-${CT_TOP_DIR}/.build}/${CT_HOST:+HOST-${CT_HOST}/}${CT_TARGET}"
CT_BUILD_DIR="${CT_BUILD_TOP_DIR}/build"
CT_PREFIX_DIR="${CT_PREFIX:-${HOME}/x-tools}/${CT_HOST:+HOST-${CT_HOST}/}${CT_TARGET}"
CT_RM_RF_PREFIX_DIR=y
CT_REMOVE_DOCS=y
CT_INSTALL_LICENSES=y
CT_PREFIX_DIR_RO=y
CT_STRIP_HOST_TOOLCHAIN_EXECUTABLES=y
# CT_STRIP_TARGET_TOOLCHAIN_EXECUTABLES is not set

#
# Downloading
#
CT_DOWNLOAD_AGENT_WGET=y
# CT_DOWNLOAD_AGENT_NONE is not set
# CT_FORBID_DOWNLOAD is not set
# CT_FORCE_DOWNLOAD is not set
CT_CONNECT_TIMEOUT=10
CT_DOWNLOAD_WGET_OPTIONS="--passive-ftp --tries=3 -nc --progress=dot:binary"
# CT_ONLY_DOWNLOAD is not set
# CT_USE_MIRROR is not set
CT_VERIFY_DOWNLOAD_DIGEST=y
CT_VERIFY_DOWNLOAD_DIGEST_SHA512=y
# CT_VERIFY_DOWNLOAD_DIGEST_SHA256 is not set
# CT_VERIFY_DOWNLOAD_DIGEST_SHA1 is not set
# CT_VERIFY_DOWNLOAD_DIGEST_MD5 is not set
CT_VERIFY_DOWNLOAD_DIGEST_ALG="sha512"
# CT_VERIFY_DOWNLOAD_SIGNATURE is not set

#
# Extracting
#
# CT_FORCE_EXTRACT is not set
CT_OVERRIDE_CONFIG_GUESS_SUB=y
# CT_ONLY_EXTRACT is not set
CT_PATCH_BUNDLED=y
# CT_PATCH_LOCAL is not set
# CT_PATCH_BUNDLED_LOCAL is not set
# CT_PATCH_LOCAL_BUNDLED is not set
# CT_PATCH_NONE is not set
CT_PATCH_ORDER="bundled"

#
# Build behavior
#
CT_PARALLEL_JOBS=8
CT_LOAD=""
CT_USE_PIPES=y
CT_EXTRA_CFLAGS_FOR_BUILD=""
CT_EXTRA_CXXFLAGS_FOR_BUILD=""
CT_EXTRA_LDFLAGS_FOR_BUILD=""
CT_EXTRA_CFLAGS_FOR_HOST=""
CT_EXTRA_LDFLAGS_FOR_HOST=""
# CT_CONFIG_SHELL_SH is not set
# CT_CONFIG_SHELL_ASH is not set
CT_CONFIG_SHELL_BASH=y
# CT_CONFIG_SHELL_CUSTOM is not set
CT_CONFIG_SHELL="${bash}"

#
# Logging
#
# CT_LOG_ERROR is not set
# CT_LOG_WARN is not set
# CT_LOG_INFO is not set
CT_LOG_EXTRA=y
# CT_LOG_ALL is not set
# CT_LOG_DEBUG is not set
CT_LOG_LEVEL_MAX="EXTRA"
# CT_LOG_SEE_TOOLS_WARN is not set
CT_LOG_PROGRESS_BAR=y
CT_LOG_TO_FILE=y
# CT_LOG_FILE_COMPRESS is not set
# end of Paths and misc options

#
# Target options
#
# CT_ARCH_ALPHA is not set
# CT_ARCH_ARC is not set
CT_ARCH_ARM=y
# CT_ARCH_AVR is not set
# CT_ARCH_C6X is not set
# CT_ARCH_M68K is not set
# CT_ARCH_MICROBLAZE is not set
# CT_ARCH_MIPS is not set
# CT_ARCH_MOXIE is not set
# CT_ARCH_MSP430 is not set
# CT_ARCH_NIOS2 is not set
# CT_ARCH_POWERPC is not set
# CT_ARCH_PRU is not set
# CT_ARCH_RISCV is not set
# CT_ARCH_S390 is not set
# CT_ARCH_SH is not set
# CT_ARCH_SPARC is not set
# CT_ARCH_X86 is not set
# CT_ARCH_XTENSA is not set
CT_ARCH="arm"
CT_ARCH_CHOICE_KSYM="ARM"
CT_ARCH_CPU=""
CT_ARCH_TUNE=""
CT_ARCH_ARM_SHOW=y

#
# Options for arm
#
CT_ARCH_ARM_PKG_KSYM=""
# CT_ARCH_ARM_MODE="thumb"
# CT_ARCH_ARM_MODE_ARM is not set
CT_ARCH_ARM_MODE_THUMB=y
# CT_ARCH_ARM_INTERWORKING is not set
CT_ARCH_ARM_EABI_FORCE=y
CT_ARCH_ARM_EABI=y
CT_ALL_ARCH_CHOICES="ALPHA ARC ARM AVR C6X M68K MICROBLAZE MIPS MOXIE MSP430 NIOS2 POWERPC PRU RISCV S390 SH SPARC X86 XTENSA"
CT_ARCH_SUFFIX=""
# CT_OMIT_TARGET_VENDOR is not set

#
# Generic target options
#
CT_MULTILIB=y
# CT_DEMULTILIB is not set
CT_ARCH_SUPPORTS_BOTH_MMU=y
CT_ARCH_DEFAULT_HAS_MMU=y
# CT_ARCH_USE_MMU is not set
CT_ARCH_SUPPORTS_FLAT_FORMAT=y
CT_ARCH_SUPPORTS_EITHER_ENDIAN=y
CT_ARCH_DEFAULT_LE=y
# CT_ARCH_BE is not set
CT_ARCH_LE=y
CT_ARCH_ENDIAN="little"
CT_ARCH_SUPPORTS_32=y
CT_ARCH_SUPPORTS_64=y
CT_ARCH_DEFAULT_32=y
CT_ARCH_BITNESS=32
CT_ARCH_32=y
# CT_ARCH_64 is not set

#
# Target optimisations
#
CT_ARCH_SUPPORTS_WITH_ARCH=y
CT_ARCH_SUPPORTS_WITH_CPU=y
CT_ARCH_SUPPORTS_WITH_TUNE=y
CT_ARCH_SUPPORTS_WITH_FLOAT=y
CT_ARCH_SUPPORTS_WITH_FPU=y
CT_ARCH_SUPPORTS_SOFTFP=y
CT_ARCH_EXCLUSIVE_WITH_CPU=y
CT_ARCH_ARCH=""
CT_ARCH_FPU=""
CT_ARCH_FLOAT_AUTO=y
# CT_ARCH_FLOAT_HW is not set
# CT_ARCH_FLOAT_SOFTFP is not set
# CT_ARCH_FLOAT_SW is not set
CT_TARGET_CFLAGS=""
CT_TARGET_LDFLAGS=""
CT_ARCH_FLOAT="auto"
# end of Target options

#
# Toolchain options
#

#
# General toolchain options
#
CT_WANTS_STATIC_LINK=y
# CT_STATIC_TOOLCHAIN is not set
CT_SHOW_CT_VERSION=y
CT_TOOLCHAIN_PKGVERSION=""
CT_TOOLCHAIN_BUGURL=""

#
# Tuple completion and aliasing
#
CT_TARGET_VENDOR="none"
CT_TARGET_ALIAS_SED_EXPR=""
CT_TARGET_ALIAS=""

#
# Toolchain type
#
# CT_NATIVE is not set
CT_CROSS=y
# CT_CROSS_NATIVE is not set
# CT_CANADIAN is not set
CT_TOOLCHAIN_TYPE="cross"

#
# Build system
#
CT_BUILD=""
CT_BUILD_PREFIX=""
CT_BUILD_SUFFIX=""

#
# Misc options
#
# CT_TOOLCHAIN_ENABLE_NLS is not set
# end of Toolchain options

#
# Operating System
#
CT_BARE_METAL=y
CT_KERNEL_BARE_METAL=y
# CT_KERNEL_LINUX is not set
CT_KERNEL="bare-metal"
CT_KERNEL_CHOICE_KSYM="BARE_METAL"
CT_KERNEL_BARE_METAL_SHOW=y

#
# Options for bare-metal
#
CT_KERNEL_BARE_METAL_PKG_KSYM=""
CT_ALL_KERNEL_CHOICES="BARE_METAL LINUX WINDOWS"

#
# Common kernel options
#
# end of Operating System

#
# Binary utilities
#
# CT_ARCH_BINFMT_ELF is not set
CT_ARCH_BINFMT_FLAT=y
# CT_ARCH_BINFMT_FDPIC is not set
CT_BINUTILS_BINUTILS=y
CT_BINUTILS="binutils"
CT_BINUTILS_CHOICE_KSYM="BINUTILS"
CT_BINUTILS_BINUTILS_SHOW=y

#
# Options for binutils
#
CT_BINUTILS_BINUTILS_PKG_KSYM="BINUTILS"
CT_BINUTILS_DIR_NAME="binutils"
CT_BINUTILS_USE_GNU=y
# CT_BINUTILS_USE_LINARO is not set
# CT_BINUTILS_USE_ORACLE is not set
CT_BINUTILS_USE="BINUTILS"
CT_BINUTILS_PKG_NAME="binutils"
CT_BINUTILS_SRC_RELEASE=y
# CT_BINUTILS_SRC_DEVEL is not set
# CT_BINUTILS_SRC_CUSTOM is not set
CT_BINUTILS_PATCH_GLOBAL=y
# CT_BINUTILS_PATCH_BUNDLED is not set
# CT_BINUTILS_PATCH_LOCAL is not set
# CT_BINUTILS_PATCH_BUNDLED_LOCAL is not set
# CT_BINUTILS_PATCH_LOCAL_BUNDLED is not set
# CT_BINUTILS_PATCH_NONE is not set
CT_BINUTILS_PATCH_ORDER="global"
CT_BINUTILS_V_2_38=y
# CT_BINUTILS_V_2_37 is not set
# CT_BINUTILS_V_2_36 is not set
# CT_BINUTILS_V_2_35 is not set
# CT_BINUTILS_V_2_34 is not set
# CT_BINUTILS_V_2_33 is not set
# CT_BINUTILS_V_2_32 is not set
# CT_BINUTILS_V_2_31 is not set
# CT_BINUTILS_V_2_30 is not set
# CT_BINUTILS_V_2_29 is not set
# CT_BINUTILS_V_2_28 is not set
# CT_BINUTILS_V_2_27 is not set
# CT_BINUTILS_V_2_26 is not set
CT_BINUTILS_VERSION="2.38"
CT_BINUTILS_MIRRORS="$(CT_Mirrors GNU binutils) $(CT_Mirrors sourceware binutils/releases)"
CT_BINUTILS_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_BINUTILS_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_BINUTILS_ARCHIVE_FORMATS=".tar.xz .tar.bz2 .tar.gz"
CT_BINUTILS_SIGNATURE_FORMAT="packed/.sig"
CT_BINUTILS_later_than_2_30=y
CT_BINUTILS_2_30_or_later=y
CT_BINUTILS_later_than_2_27=y
CT_BINUTILS_2_27_or_later=y
CT_BINUTILS_later_than_2_26=y
CT_BINUTILS_2_26_or_later=y

#
# GNU binutils
#
CT_BINUTILS_GOLD_SUPPORTS_ARCH=y
CT_BINUTILS_GOLD_SUPPORT=y
CT_BINUTILS_FORCE_LD_BFD_ONLY=y
CT_BINUTILS_LINKER_LD=y
CT_BINUTILS_LINKERS_LIST="ld"
CT_BINUTILS_LINKER_DEFAULT="bfd"
CT_BINUTILS_PLUGINS=y
CT_BINUTILS_RELRO=m
CT_BINUTILS_DETERMINISTIC_ARCHIVES=y
CT_BINUTILS_EXTRA_CONFIG_ARRAY="--enable-initfini-array --disable-nls --without-x --disable-gdbtk --without-tcl --without-tk --enable-plugins --disable-gdb --without-gdb"

#
# elf2flt
#
CT_ELF2FLT_REQUIRES=y
CT_ELF2FLT_DIR_NAME="elf2flt"
CT_ELF2FLT_PKG_NAME="elf2flt"
CT_ELF2FLT_SRC_DEVEL=y
# CT_ELF2FLT_SRC_CUSTOM is not set
CT_ELF2FLT_DEVEL_VCS_git=y
# CT_ELF2FLT_DEVEL_VCS_svn is not set
# CT_ELF2FLT_DEVEL_VCS_hg is not set
# CT_ELF2FLT_DEVEL_VCS_cvs is not set
CT_ELF2FLT_DEVEL_VCS="git"
CT_ELF2FLT_DEVEL_URL="https://github.com/uclinux-dev/elf2flt.git"
CT_ELF2FLT_DEVEL_BRANCH=""
CT_ELF2FLT_DEVEL_REVISION="453398f917d167f8c308c8f997270c48ae8f8b12"
CT_ELF2FLT_DEVEL_SUBDIR=""
CT_ELF2FLT_DEVEL_BOOTSTRAP=""
CT_ELF2FLT_PATCH_GLOBAL=y
# CT_ELF2FLT_PATCH_BUNDLED is not set
# CT_ELF2FLT_PATCH_LOCAL is not set
# CT_ELF2FLT_PATCH_BUNDLED_LOCAL is not set
# CT_ELF2FLT_PATCH_LOCAL_BUNDLED is not set
# CT_ELF2FLT_PATCH_NONE is not set
CT_ELF2FLT_PATCH_ORDER="global"
CT_ELF2FLT_VERSION="unknown"
CT_ELF2FLT_EXTRA_CONFIG_ARRAY=""
CT_ALL_BINUTILS_CHOICES="BINUTILS"
# end of Binary utilities

#
# C-library
#
CT_LIBC_NEWLIB=y
# CT_LIBC_NONE is not set
CT_LIBC="newlib"
CT_LIBC_CHOICE_KSYM="NEWLIB"
CT_THREADS="none"
CT_LIBC_NEWLIB_SHOW=y

#
# Options for newlib
#
CT_LIBC_NEWLIB_PKG_KSYM="NEWLIB"
CT_NEWLIB_DIR_NAME="newlib"
CT_NEWLIB_PKG_NAME="newlib"
CT_NEWLIB_SRC_RELEASE=y
# CT_NEWLIB_SRC_DEVEL is not set
# CT_NEWLIB_SRC_CUSTOM is not set
CT_NEWLIB_PATCH_GLOBAL=y
# CT_NEWLIB_PATCH_BUNDLED is not set
# CT_NEWLIB_PATCH_LOCAL is not set
# CT_NEWLIB_PATCH_BUNDLED_LOCAL is not set
# CT_NEWLIB_PATCH_LOCAL_BUNDLED is not set
# CT_NEWLIB_PATCH_NONE is not set
CT_NEWLIB_PATCH_ORDER="global"
CT_NEWLIB_V_4_1=y
# CT_NEWLIB_V_3_3 is not set
# CT_NEWLIB_V_3_2 is not set
# CT_NEWLIB_V_3_1 is not set
# CT_NEWLIB_V_3_0 is not set
# CT_NEWLIB_V_2_5 is not set
CT_NEWLIB_VERSION="4.1.0"
CT_NEWLIB_MIRRORS="ftp://sourceware.org/pub/newlib"
CT_NEWLIB_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_NEWLIB_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_NEWLIB_ARCHIVE_FORMATS=".tar.gz"
CT_NEWLIB_SIGNATURE_FORMAT=""
CT_LIBC_NEWLIB_TARGET_CFLAGS=""
# CT_LIBC_NEWLIB_IO_C99FMT is not set
# CT_LIBC_NEWLIB_IO_LL is not set
# CT_LIBC_NEWLIB_IO_FLOAT is not set
# CT_LIBC_NEWLIB_IO_POS_ARGS is not set
# CT_LIBC_NEWLIB_FVWRITE_IN_STREAMIO is not set
# CT_LIBC_NEWLIB_UNBUF_STREAM_OPT is not set
# CT_LIBC_NEWLIB_FSEEK_OPTIMIZATION is not set
CT_LIBC_NEWLIB_DISABLE_SUPPLIED_SYSCALLS=y
# CT_LIBC_NEWLIB_REGISTER_FINI is not set
# CT_LIBC_NEWLIB_ATEXIT_DYNAMIC_ALLOC is not set
# CT_LIBC_NEWLIB_GLOBAL_ATEXIT is not set
CT_LIBC_NEWLIB_LITE_EXIT=y
CT_LIBC_NEWLIB_REENT_SMALL=y
# CT_LIBC_NEWLIB_MULTITHREAD is not set
# CT_LIBC_NEWLIB_RETARGETABLE_LOCKING is not set
CT_LIBC_NEWLIB_EXTRA_SECTIONS=y
# CT_LIBC_NEWLIB_WIDE_ORIENT is not set
CT_LIBC_NEWLIB_ENABLE_TARGET_OPTSPACE=y
# CT_LIBC_NEWLIB_LTO is not set
CT_LIBC_NEWLIB_NANO_MALLOC=y
CT_LIBC_NEWLIB_NANO_FORMATTED_IO=y
CT_LIBC_NEWLIB_EXTRA_CONFIG_ARRAY=""
CT_ALL_LIBC_CHOICES="AVR_LIBC BIONIC GLIBC MINGW_W64 MOXIEBOX MUSL NEWLIB NONE UCLIBC_NG"
CT_LIBC_SUPPORT_THREADS_NONE=y
CT_LIBC_PROVIDES_CXA_ATEXIT=y

#
# Common C library options
#
CT_THREADS_NONE=y
# end of C-library

#
# C compiler
#
CT_CC_CORE_NEEDED=y
CT_CC_SUPPORT_CXX=y
CT_CC_SUPPORT_FORTRAN=y
CT_CC_SUPPORT_ADA=y
CT_CC_SUPPORT_OBJC=y
CT_CC_SUPPORT_OBJCXX=y
CT_CC_SUPPORT_GOLANG=y
CT_CC_GCC=y
CT_CC="gcc"
CT_CC_CHOICE_KSYM="GCC"
CT_CC_GCC_SHOW=y

#
# Options for gcc
#
CT_CC_GCC_PKG_KSYM="GCC"
CT_GCC_DIR_NAME="gcc"
CT_GCC_USE_GNU=y
# CT_GCC_USE_LINARO is not set
CT_GCC_USE="GCC"
CT_GCC_PKG_NAME="gcc"
CT_GCC_SRC_RELEASE=y
# CT_GCC_SRC_DEVEL is not set
# CT_GCC_SRC_CUSTOM is not set
CT_GCC_PATCH_GLOBAL=y
# CT_GCC_PATCH_BUNDLED is not set
# CT_GCC_PATCH_LOCAL is not set
# CT_GCC_PATCH_BUNDLED_LOCAL is not set
# CT_GCC_PATCH_LOCAL_BUNDLED is not set
# CT_GCC_PATCH_NONE is not set
CT_GCC_PATCH_ORDER="global"
CT_GCC_V_11=y
# CT_GCC_V_10 is not set
# CT_GCC_V_9 is not set
# CT_GCC_V_8 is not set
# CT_GCC_V_7 is not set
# CT_GCC_V_6 is not set
# CT_GCC_V_5 is not set
# CT_GCC_V_4_9 is not set
CT_GCC_VERSION="11.2.0"
CT_GCC_MIRRORS="$(CT_Mirrors GNU gcc/gcc-${CT_GCC_VERSION}) $(CT_Mirrors sourceware gcc/releases/gcc-${CT_GCC_VERSION})"
CT_GCC_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_GCC_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_GCC_ARCHIVE_FORMATS=".tar.xz .tar.gz"
CT_GCC_SIGNATURE_FORMAT=""
CT_GCC_later_than_11=y
CT_GCC_11_or_later=y
CT_GCC_later_than_10=y
CT_GCC_10_or_later=y
CT_GCC_later_than_9=y
CT_GCC_9_or_later=y
CT_GCC_later_than_8=y
CT_GCC_8_or_later=y
CT_GCC_later_than_7=y
CT_GCC_7_or_later=y
CT_GCC_later_than_6=y
CT_GCC_6_or_later=y
CT_GCC_later_than_5=y
CT_GCC_5_or_later=y
CT_GCC_later_than_4_9=y
CT_GCC_4_9_or_later=y
CT_CC_GCC_ENABLE_PLUGINS=y
CT_CC_GCC_HAS_LIBMPX=y
CT_CC_GCC_ENABLE_CXX_FLAGS=""
CT_CC_GCC_CORE_EXTRA_CONFIG_ARRAY=""
CT_CC_GCC_EXTRA_CONFIG_ARRAY=""
CT_CC_GCC_MULTILIB_LIST="rmprofile"
# CT_CC_GCC_STATIC_LIBSTDCXX is not set
# CT_CC_GCC_SYSTEM_ZLIB is not set
# CT_CC_GCC_CONFIG_TLS is not set

#
# Optimisation features
#
CT_CC_GCC_USE_GRAPHITE=y
CT_CC_GCC_USE_LTO=y
CT_CC_GCC_LTO_ZSTD=m

#
# Settings for libraries running on target
#
# CT_CC_GCC_ENABLE_TARGET_OPTSPACE is not set
# CT_CC_GCC_LIBMUDFLAP is not set
# CT_CC_GCC_LIBSSP is not set
# CT_CC_GCC_LIBQUADMATH is not set

#
# Misc. obscure options.
#
# CT_CC_CXA_ATEXIT is not set
CT_CC_GCC_TM_CLONE_REGISTRY=m
CT_CC_GCC_DISABLE_PCH=y
# CT_CC_GCC_LDBL_128 is not set
CT_CC_GCC_BUILD_ID=y
CT_CC_GCC_LNK_HASH_STYLE_DEFAULT=y
# CT_CC_GCC_LNK_HASH_STYLE_SYSV is not set
# CT_CC_GCC_LNK_HASH_STYLE_GNU is not set
# CT_CC_GCC_LNK_HASH_STYLE_BOTH is not set
CT_CC_GCC_LNK_HASH_STYLE=""
CT_CC_GCC_DEC_FLOATS_AUTO=y
# CT_CC_GCC_DEC_FLOATS_BID is not set
# CT_CC_GCC_DEC_FLOATS_DPD is not set
# CT_CC_GCC_DEC_FLOATS_NO is not set
CT_CC_GCC_DEC_FLOATS=""
CT_ALL_CC_CHOICES="GCC"

#
# Additional supported languages:
#
CT_CC_LANG_CXX=y
# CT_CC_LANG_FORTRAN is not set
CT_CC_LANG_OTHERS=""
# end of C compiler

#
# Debug facilities
#
CT_DEBUG_GDB=y
CT_DEBUG_GDB_PKG_KSYM="GDB"
CT_GDB_DIR_NAME="gdb"
CT_GDB_PKG_NAME="gdb"
CT_GDB_SRC_RELEASE=y
# CT_GDB_SRC_DEVEL is not set
# CT_GDB_SRC_CUSTOM is not set
CT_GDB_PATCH_GLOBAL=y
# CT_GDB_PATCH_BUNDLED is not set
# CT_GDB_PATCH_LOCAL is not set
# CT_GDB_PATCH_BUNDLED_LOCAL is not set
# CT_GDB_PATCH_LOCAL_BUNDLED is not set
# CT_GDB_PATCH_NONE is not set
CT_GDB_PATCH_ORDER="global"
CT_GDB_V_11=y
# CT_GDB_V_10 is not set
# CT_GDB_V_9 is not set
# CT_GDB_V_8_3 is not set
CT_GDB_VERSION="11.2"
CT_GDB_MIRRORS="$(CT_Mirrors GNU gdb) $(CT_Mirrors sourceware gdb/releases)"
CT_GDB_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_GDB_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_GDB_ARCHIVE_FORMATS=".tar.xz .tar.gz"
CT_GDB_SIGNATURE_FORMAT=""
CT_GDB_later_than_11=y
CT_GDB_11_or_later=y
CT_GDB_later_than_10=y
CT_GDB_10_or_later=y
CT_GDB_later_than_8_3=y
CT_GDB_8_3_or_later=y
CT_GDB_later_than_8_0=y
CT_GDB_8_0_or_later=y
CT_GDB_later_than_7_12=y
CT_GDB_7_12_or_later=y
CT_GDB_later_than_7_11=y
CT_GDB_7_11_or_later=y
CT_GDB_CROSS=y
CT_GDB_CROSS_STATIC=y
# CT_GDB_CROSS_SIM is not set
CT_GDB_CROSS_EXTRA_CONFIG_ARRAY=""

#
# In bare-metal, you'll need to
#

#
# provide your own gdbserver stub.
#
CT_GDB_GDBSERVER_TOPLEVEL=y
# CT_DEBUG_LTRACE is not set
# CT_DEBUG_STRACE is not set
CT_ALL_DEBUG_CHOICES="DUMA GDB LTRACE STRACE"
# end of Debug facilities

#
# Companion libraries
#
# CT_COMPLIBS_CHECK is not set
# CT_COMP_LIBS_CLOOG is not set
CT_COMP_LIBS_EXPAT=y
CT_COMP_LIBS_EXPAT_PKG_KSYM="EXPAT"
CT_EXPAT_DIR_NAME="expat"
CT_EXPAT_PKG_NAME="expat"
CT_EXPAT_SRC_RELEASE=y
# CT_EXPAT_SRC_DEVEL is not set
# CT_EXPAT_SRC_CUSTOM is not set
CT_EXPAT_PATCH_GLOBAL=y
# CT_EXPAT_PATCH_BUNDLED is not set
# CT_EXPAT_PATCH_LOCAL is not set
# CT_EXPAT_PATCH_BUNDLED_LOCAL is not set
# CT_EXPAT_PATCH_LOCAL_BUNDLED is not set
# CT_EXPAT_PATCH_NONE is not set
CT_EXPAT_PATCH_ORDER="global"
CT_EXPAT_V_2_4=y
CT_EXPAT_VERSION="2.4.1"
CT_EXPAT_MIRRORS="http://downloads.sourceforge.net/project/expat/expat/${CT_EXPAT_VERSION} https://github.com/libexpat/libexpat/releases/download/R_${CT_EXPAT_VERSION//./_}"
CT_EXPAT_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_EXPAT_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_EXPAT_ARCHIVE_FORMATS=".tar.xz .tar.lz .tar.bz2 .tar.gz"
CT_EXPAT_SIGNATURE_FORMAT=""
# CT_COMP_LIBS_GETTEXT is not set
CT_COMP_LIBS_GMP=y
CT_COMP_LIBS_GMP_PKG_KSYM="GMP"
CT_GMP_DIR_NAME="gmp"
CT_GMP_PKG_NAME="gmp"
CT_GMP_SRC_RELEASE=y
# CT_GMP_SRC_DEVEL is not set
# CT_GMP_SRC_CUSTOM is not set
CT_GMP_PATCH_GLOBAL=y
# CT_GMP_PATCH_BUNDLED is not set
# CT_GMP_PATCH_LOCAL is not set
# CT_GMP_PATCH_BUNDLED_LOCAL is not set
# CT_GMP_PATCH_LOCAL_BUNDLED is not set
# CT_GMP_PATCH_NONE is not set
CT_GMP_PATCH_ORDER="global"
CT_GMP_V_6_2=y
# CT_GMP_V_6_1 is not set
CT_GMP_VERSION="6.2.1"
CT_GMP_MIRRORS="https://gmplib.org/download/gmp https://gmplib.org/download/gmp/archive $(CT_Mirrors GNU gmp)"
CT_GMP_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_GMP_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_GMP_ARCHIVE_FORMATS=".tar.xz .tar.lz .tar.bz2"
CT_GMP_SIGNATURE_FORMAT="packed/.sig"
CT_COMP_LIBS_ISL=y
CT_COMP_LIBS_ISL_PKG_KSYM="ISL"
CT_ISL_DIR_NAME="isl"
CT_ISL_PKG_NAME="isl"
CT_ISL_SRC_RELEASE=y
# CT_ISL_SRC_DEVEL is not set
# CT_ISL_SRC_CUSTOM is not set
CT_ISL_PATCH_GLOBAL=y
# CT_ISL_PATCH_BUNDLED is not set
# CT_ISL_PATCH_LOCAL is not set
# CT_ISL_PATCH_BUNDLED_LOCAL is not set
# CT_ISL_PATCH_LOCAL_BUNDLED is not set
# CT_ISL_PATCH_NONE is not set
CT_ISL_PATCH_ORDER="global"
CT_ISL_V_0_24=y
# CT_ISL_V_0_23 is not set
# CT_ISL_V_0_22 is not set
# CT_ISL_V_0_21 is not set
# CT_ISL_V_0_20 is not set
# CT_ISL_V_0_19 is not set
# CT_ISL_V_0_18 is not set
# CT_ISL_V_0_17 is not set
# CT_ISL_V_0_16 is not set
# CT_ISL_V_0_15 is not set
CT_ISL_VERSION="0.24"
CT_ISL_MIRRORS="https://libisl.sourceforge.io"
CT_ISL_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_ISL_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_ISL_ARCHIVE_FORMATS=".tar.xz .tar.bz2 .tar.gz"
CT_ISL_SIGNATURE_FORMAT=""
CT_ISL_later_than_0_18=y
CT_ISL_0_18_or_later=y
CT_ISL_later_than_0_15=y
CT_ISL_0_15_or_later=y
# CT_COMP_LIBS_LIBELF is not set
CT_COMP_LIBS_LIBICONV=y
CT_COMP_LIBS_LIBICONV_PKG_KSYM="LIBICONV"
CT_LIBICONV_DIR_NAME="libiconv"
CT_LIBICONV_PKG_NAME="libiconv"
CT_LIBICONV_SRC_RELEASE=y
# CT_LIBICONV_SRC_DEVEL is not set
# CT_LIBICONV_SRC_CUSTOM is not set
CT_LIBICONV_PATCH_GLOBAL=y
# CT_LIBICONV_PATCH_BUNDLED is not set
# CT_LIBICONV_PATCH_LOCAL is not set
# CT_LIBICONV_PATCH_BUNDLED_LOCAL is not set
# CT_LIBICONV_PATCH_LOCAL_BUNDLED is not set
# CT_LIBICONV_PATCH_NONE is not set
CT_LIBICONV_PATCH_ORDER="global"
CT_LIBICONV_V_1_16=y
# CT_LIBICONV_V_1_15 is not set
CT_LIBICONV_VERSION="1.16"
CT_LIBICONV_MIRRORS="$(CT_Mirrors GNU libiconv)"
CT_LIBICONV_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_LIBICONV_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_LIBICONV_ARCHIVE_FORMATS=".tar.gz"
CT_LIBICONV_SIGNATURE_FORMAT="packed/.sig"
CT_COMP_LIBS_MPC=y
CT_COMP_LIBS_MPC_PKG_KSYM="MPC"
CT_MPC_DIR_NAME="mpc"
CT_MPC_PKG_NAME="mpc"
CT_MPC_SRC_RELEASE=y
# CT_MPC_SRC_DEVEL is not set
# CT_MPC_SRC_CUSTOM is not set
CT_MPC_PATCH_GLOBAL=y
# CT_MPC_PATCH_BUNDLED is not set
# CT_MPC_PATCH_LOCAL is not set
# CT_MPC_PATCH_BUNDLED_LOCAL is not set
# CT_MPC_PATCH_LOCAL_BUNDLED is not set
# CT_MPC_PATCH_NONE is not set
CT_MPC_PATCH_ORDER="global"
CT_MPC_V_1_2=y
# CT_MPC_V_1_1 is not set
# CT_MPC_V_1_0 is not set
CT_MPC_VERSION="1.2.1"
CT_MPC_MIRRORS="http://www.multiprecision.org/downloads $(CT_Mirrors GNU mpc)"
CT_MPC_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_MPC_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_MPC_ARCHIVE_FORMATS=".tar.gz"
CT_MPC_SIGNATURE_FORMAT="packed/.sig"
CT_MPC_later_than_1_1_0=y
CT_MPC_1_1_0_or_later=y
CT_COMP_LIBS_MPFR=y
CT_COMP_LIBS_MPFR_PKG_KSYM="MPFR"
CT_MPFR_DIR_NAME="mpfr"
CT_MPFR_PKG_NAME="mpfr"
CT_MPFR_SRC_RELEASE=y
# CT_MPFR_SRC_DEVEL is not set
# CT_MPFR_SRC_CUSTOM is not set
CT_MPFR_PATCH_GLOBAL=y
# CT_MPFR_PATCH_BUNDLED is not set
# CT_MPFR_PATCH_LOCAL is not set
# CT_MPFR_PATCH_BUNDLED_LOCAL is not set
# CT_MPFR_PATCH_LOCAL_BUNDLED is not set
# CT_MPFR_PATCH_NONE is not set
CT_MPFR_PATCH_ORDER="global"
CT_MPFR_V_4_1=y
# CT_MPFR_V_4_0 is not set
# CT_MPFR_V_3_1 is not set
CT_MPFR_VERSION="4.1.0"
CT_MPFR_MIRRORS="http://www.mpfr.org/mpfr-${CT_MPFR_VERSION} $(CT_Mirrors GNU mpfr)"
CT_MPFR_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_MPFR_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_MPFR_ARCHIVE_FORMATS=".tar.xz .tar.bz2 .tar.gz .zip"
CT_MPFR_SIGNATURE_FORMAT="packed/.asc"
CT_MPFR_later_than_4_0_0=y
CT_MPFR_4_0_0_or_later=y
CT_COMP_LIBS_NCURSES=y
CT_COMP_LIBS_NCURSES_PKG_KSYM="NCURSES"
CT_NCURSES_DIR_NAME="ncurses"
CT_NCURSES_PKG_NAME="ncurses"
CT_NCURSES_SRC_RELEASE=y
# CT_NCURSES_SRC_DEVEL is not set
# CT_NCURSES_SRC_CUSTOM is not set
CT_NCURSES_PATCH_GLOBAL=y
# CT_NCURSES_PATCH_BUNDLED is not set
# CT_NCURSES_PATCH_LOCAL is not set
# CT_NCURSES_PATCH_BUNDLED_LOCAL is not set
# CT_NCURSES_PATCH_LOCAL_BUNDLED is not set
# CT_NCURSES_PATCH_NONE is not set
CT_NCURSES_PATCH_ORDER="global"
CT_NCURSES_V_6_2=y
# CT_NCURSES_V_6_1 is not set
# CT_NCURSES_V_6_0 is not set
CT_NCURSES_VERSION="6.2"
CT_NCURSES_MIRRORS="https://invisible-mirror.net/archives/ncurses $(CT_Mirrors GNU ncurses)"
CT_NCURSES_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_NCURSES_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_NCURSES_ARCHIVE_FORMATS=".tar.gz"
CT_NCURSES_SIGNATURE_FORMAT="packed/.sig"
CT_NCURSES_NEW_ABI=y
CT_NCURSES_HOST_CONFIG_ARGS=""
CT_NCURSES_HOST_DISABLE_DB=y
CT_NCURSES_HOST_FALLBACKS="linux,xterm,xterm-color,xterm-256color,vt100"
CT_NCURSES_TARGET_CONFIG_ARGS=""
# CT_NCURSES_TARGET_DISABLE_DB is not set
CT_NCURSES_TARGET_FALLBACKS=""
CT_COMP_LIBS_NEWLIB_NANO=y
CT_COMP_LIBS_NEWLIB_NANO_PKG_KSYM="NEWLIB_NANO"
CT_NEWLIB_NANO_DIR_NAME="newlib-nano"
CT_NEWLIB_NANO_PKG_NAME="newlib-nano"
CT_NEWLIB_NANO_SRC_RELEASE=y
# CT_NEWLIB_NANO_SRC_DEVEL is not set
# CT_NEWLIB_NANO_SRC_CUSTOM is not set
CT_NEWLIB_NANO_PATCH_GLOBAL=y
# CT_NEWLIB_NANO_PATCH_BUNDLED is not set
# CT_NEWLIB_NANO_PATCH_LOCAL is not set
# CT_NEWLIB_NANO_PATCH_BUNDLED_LOCAL is not set
# CT_NEWLIB_NANO_PATCH_LOCAL_BUNDLED is not set
# CT_NEWLIB_NANO_PATCH_NONE is not set
CT_NEWLIB_NANO_PATCH_ORDER="global"
CT_NEWLIB_NANO_V_4_1=y
# CT_NEWLIB_NANO_V_3_3 is not set
CT_NEWLIB_NANO_VERSION="4.1.0"
CT_NEWLIB_NANO_MIRRORS="ftp://sourceware.org/pub/newlib"
CT_NEWLIB_NANO_ARCHIVE_FILENAME="newlib-@{version}"
CT_NEWLIB_NANO_ARCHIVE_DIRNAME="newlib-@{version}"
CT_NEWLIB_NANO_ARCHIVE_FORMATS=".tar.gz"
CT_NEWLIB_NANO_SIGNATURE_FORMAT=""
CT_NEWLIB_NANO_GCC_LIBSTDCXX=y
CT_NEWLIB_NANO_GCC_LIBSTDCXX_TARGET_CXXFLAGS="-fno-exceptions"
CT_NEWLIB_NANO_INSTALL_IN_TARGET=y
CT_LIBC_NEWLIB_NANO_TARGET_CFLAGS=""
# CT_LIBC_NEWLIB_NANO_IO_C99FMT is not set
# CT_LIBC_NEWLIB_NANO_IO_LL is not set
# CT_LIBC_NEWLIB_NANO_IO_FLOAT is not set
# CT_LIBC_NEWLIB_NANO_IO_POS_ARGS is not set
# CT_LIBC_NEWLIB_NANO_FVWRITE_IN_STREAMIO is not set
# CT_LIBC_NEWLIB_NANO_UNBUF_STREAM_OPT is not set
# CT_LIBC_NEWLIB_NANO_FSEEK_OPTIMIZATION is not set
CT_LIBC_NEWLIB_NANO_DISABLE_SUPPLIED_SYSCALLS=y
# CT_LIBC_NEWLIB_NANO_REGISTER_FINI is not set
# CT_LIBC_NEWLIB_NANO_ATEXIT_DYNAMIC_ALLOC is not set
# CT_LIBC_NEWLIB_NANO_GLOBAL_ATEXIT is not set
CT_LIBC_NEWLIB_NANO_LITE_EXIT=y
CT_LIBC_NEWLIB_NANO_REENT_SMALL=y
# CT_LIBC_NEWLIB_NANO_MULTITHREAD is not set
# CT_LIBC_NEWLIB_NANO_RETARGETABLE_LOCKING is not set
CT_LIBC_NEWLIB_NANO_EXTRA_SECTIONS=y
# CT_LIBC_NEWLIB_NANO_WIDE_ORIENT is not set
CT_LIBC_NEWLIB_NANO_ENABLE_TARGET_OPTSPACE=y
# CT_LIBC_NEWLIB_NANO_LTO is not set
CT_LIBC_NEWLIB_NANO_NANO_MALLOC=y
CT_LIBC_NEWLIB_NANO_NANO_FORMATTED_IO=y
CT_LIBC_NEWLIB_NANO_EXTRA_CONFIG_ARRAY=""
CT_COMP_LIBS_ZLIB=y
CT_COMP_LIBS_ZLIB_PKG_KSYM="ZLIB"
CT_ZLIB_DIR_NAME="zlib"
CT_ZLIB_PKG_NAME="zlib"
CT_ZLIB_SRC_RELEASE=y
# CT_ZLIB_SRC_DEVEL is not set
# CT_ZLIB_SRC_CUSTOM is not set
CT_ZLIB_PATCH_GLOBAL=y
# CT_ZLIB_PATCH_BUNDLED is not set
# CT_ZLIB_PATCH_LOCAL is not set
# CT_ZLIB_PATCH_BUNDLED_LOCAL is not set
# CT_ZLIB_PATCH_LOCAL_BUNDLED is not set
# CT_ZLIB_PATCH_NONE is not set
CT_ZLIB_PATCH_ORDER="global"
CT_ZLIB_V_1_2_11=y
CT_ZLIB_VERSION="1.2.11"
CT_ZLIB_MIRRORS="http://downloads.sourceforge.net/project/libpng/zlib/${CT_ZLIB_VERSION} https://www.zlib.net/"
CT_ZLIB_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
CT_ZLIB_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
CT_ZLIB_ARCHIVE_FORMATS=".tar.xz .tar.gz"
CT_ZLIB_SIGNATURE_FORMAT="packed/.asc"
CT_ALL_COMP_LIBS_CHOICES="CLOOG EXPAT GETTEXT GMP GNUPRUMCU ISL LIBELF LIBICONV MPC MPFR NCURSES NEWLIB_NANO PICOLIBC ZLIB"
CT_LIBICONV_NEEDED=y
# CT_GETTEXT_NEEDED is not set
CT_GMP_NEEDED=y
CT_MPFR_NEEDED=y
CT_ISL_NEEDED=y
CT_MPC_NEEDED=y
CT_EXPAT_NEEDED=y
CT_NCURSES_NEEDED=y
CT_ZLIB_NEEDED=y
CT_LIBICONV=y
CT_GMP=y
CT_MPFR=y
CT_ISL=y
CT_MPC=y
CT_EXPAT=y
CT_NCURSES=y
CT_ZLIB=y
# end of Companion libraries

#
# Companion tools
#
# CT_COMP_TOOLS_FOR_HOST is not set
# CT_COMP_TOOLS_AUTOCONF is not set
# CT_COMP_TOOLS_AUTOMAKE is not set
# CT_COMP_TOOLS_BISON is not set
# CT_COMP_TOOLS_DTC is not set
# CT_COMP_TOOLS_LIBTOOL is not set
# CT_COMP_TOOLS_M4 is not set
# CT_COMP_TOOLS_MAKE is not set
CT_ALL_COMP_TOOLS_CHOICES="AUTOCONF AUTOMAKE BISON DTC LIBTOOL M4 MAKE"
# end of Companion tools

#
# Test suite
#
# CT_TEST_SUITE_GCC is not set
# end of Test suite
```

</details>

--- a/config/arch/arm.in
+++ b/config/arch/arm.in
@@ -9,13 +9,13 @@
 ## select ARCH_SUPPORTS_FLAT_FORMAT
 ## select ARCH_SUPPORTS_EITHER_ENDIAN
 ## select ARCH_DEFAULT_LE
-## select ARCH_SUPPORTS_WITH_ARCH
-## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_ARCH if !(MULTILIB && ARCH_32)
+## select ARCH_SUPPORTS_WITH_CPU if !(MULTILIB && ARCH_32)
 ## select ARCH_EXCLUSIVE_WITH_CPU
-## select ARCH_SUPPORTS_WITH_TUNE
-## select ARCH_SUPPORTS_WITH_FLOAT if ARCH_32
-## select ARCH_SUPPORTS_WITH_FPU if ARCH_32
-## select ARCH_SUPPORTS_SOFTFP if ARCH_32
+## select ARCH_SUPPORTS_WITH_TUNE if !(MULTILIB && ARCH_32)
+## select ARCH_SUPPORTS_WITH_FLOAT if ARCH_32 && !MULTILIB
+## select ARCH_SUPPORTS_WITH_FPU if ARCH_32 && !MULTILIB
+## select ARCH_SUPPORTS_SOFTFP if ARCH_32 && !MULTILIB
 ## select LINUX_REQUIRE_3_7_or_later if ARCH_64 && KERNEL_LINUX
 
 ## help The ARM architecture, as defined by:
@@ -26,11 +26,13 @@ config ARCH_ARM_MODE
     string
     default "arm"   if ARCH_ARM_MODE_ARM
     default "thumb" if ARCH_ARM_MODE_THUMB
+    depends on !MULTILIB
 
 choice
     bool
     prompt "Default instruction set mode"
     default ARCH_ARM_MODE_ARM
+    depends on !MULTILIB
 
 config ARCH_ARM_MODE_ARM
     bool
@@ -49,6 +51,7 @@ endchoice
 config ARCH_ARM_INTERWORKING
     bool
     prompt "Use Thumb-interworking (READ HELP)"
+    depends on !MULTILIB
     help
       Excerpt from the gcc manual:
       

--- a/config/cc/gcc.in
+++ b/config/cc/gcc.in
@@ -80,6 +80,7 @@ config CC_GCC_MULTILIB_LIST
     string "List of multilib variants"
     depends on MULTILIB
     default "m2,m2e,m4,m4-single,m4-single-only,m2a,m2a-single" if GCC_11_or_later && ARCH_SH
+    default "aprofile,rmprofile" if ARCH_ARM && ARCH_32
     help
       Architecture-specific option of expanding or restricting the list of
       the multilib variants to be built. Refer to GCC installation manual

